packages/ckeditor5-ui/src/button/buttonview.ts

```diff
@@ -209,10 +233,13 @@ export default class ButtonView extends View<HTMLButtonElement> implements Butto
 					bind.if( 'withText', 'ck-button_with-text' ),
 					bind.if( 'withKeystroke', 'ck-button_with-keystroke' )
 				],
+				role: bind.to( 'role' ),
 				type: bind.to( 'type', value => value ? value : 'button' ),
 				tabindex: bind.to( 'tabindex' ),
-				'aria-labelledby': `ck-editor__aria-label_${ ariaLabelUid }`,
+				'aria-label': bind.to( 'ariaLabel' ),
+				'aria-labelledby': bind.to( 'ariaLabelledBy' ),
```

Comment:
The existing test checks if `aria-labelledby` starts with `ck-editor__aria-label`. This is a default value but we also need a test to see if the new observable `ariaLabelledBy` property is actually observable and what it does.

Reply:
Test added.